index.html

```diff
@@ -0,0 +1,122 @@
+      content="text/html; charset=utf-8"
+    />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>판다마켓</title>
+    <link rel="stylesheet" href="css/reset.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="stylesheet" href="css/index.css" />
+  </head>
+  <body>
+    <div class="wrap">
+      <header>
+        <a href="./" class="logo">
+          <img src="/images/logo.svg" alt="판다마켓" />
+          판다마켓
+        </a>
+        <a class="login" href="./pages/signin.html">로그인</a>
+      </header>
+      <main>
+        <div class="visual">
+          <img src="images/panda_top.svg" class="visual__bg" alt="" />
+          <div class="visual__content">
+            <h1>
+              일상의 모든 물건을<br />
+              거래해 보세요
+            </h1>
+            <a class="goShopping" href="./pages/items.html">구경하러 가기</a>
+          </div>
+        </div>
+        <section class="item">
+          <div class="item__image image01"></div>
+          <div class="item__text">
+            <h2 class="item__maintitle">
+              인기 상품을<br />
+              확인해 보세요
+            </h2>
+            <div class="item__subtitle">Hot item</div>
+            <div class="item__desc">
+              가장 HOT한 중고거래 물품을<br />
+              판다 마켓에서 확인해 보세요
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="item__image image02"></div>
+          <div class="item__text">
+            <h2 class="item__maintitle">
+              구매를 원하는<br />
+              상품을 검색하세요
+            </h2>
+            <div class="item__subtitle">Search</div>
+            <div class="item__desc">
+              구매하고 싶은 물품은 검색해서 <br />
+              쉽게 찾아보세요
+            </div>
+          </div>
+        </section>
+        <section class="item">
+          <div class="item__image image03"></div>
+          <div class="item__text">
+            <h2 class="item__maintitle">
+              판매를 원하는<br />
+              상품을 등록하세요
+            </h2>
+            <div class="item__subtitle">Register</div>
+            <div class="item__desc">
+              어떤 물건이든 판매하고 싶은 상품을<br />
+              쉽게 등록하세요
+            </div>
+          </div>
+        </section>
+        <div class="visual">
+          <div class="visual__content">
+            <h1>
+              믿을 수 있는<br />
+              판다마켓 중고거래
+            </h1>
+          </div>
+          <img src="images/panda_bottom.svg" class="visual__bg bottom" alt="" />
+        </div>
+      </main>
+      <footer>
+        <div class="footer__copyright">© codeit - 2024</div>
+        <div class="footer__FAQ">
+          <a href="#">Privacy Policy</a>
+          <a href="#">FAQ</a>
+        </div>
+        <div class="socials">
+          <a
+            href="https://www.facebook.com/"
+            class="facebook"
+            aria-label="facebook"
+            target="_blank"
+          ></a>
```

Comment:
요런 부분은 self-closing 태그를 사용할 수도 있답니당! 
```suggestion
          <a
            href="https://www.facebook.com/"
            class="facebook"
            aria-label="facebook"
            target="_blank"
          />
```